stdlib/public/core/Policy.swift

```diff
@@ -382,7 +382,7 @@ precedencegroup BitwiseShiftPrecedence {
 // Standard postfix operators.
 postfix operator ++
 postfix operator --
-postfix operator ... : Comparable
+postfix operator ...: Comparable
```

Comment:
This section with the operators is the only place where I prefer having a space before the colon, but it's probably not worth the exception. Anyone else?

Otherwise LGTM! Thanks @twostraws!